services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -174,6 +197,31 @@ async function analyzeTable(table, config) {
   };
 }
 
+function createBelongsToReference(referenceTable, tableReferences, constraint) {
```

Comment:
This function is created to reduce cognitive complexity of `createAllReferences`.